Source/Core/VideoCommon/CommandProcessor.cpp

```diff
@@ -140,52 +140,63 @@ void Init()
 
 void RegisterMMIO(MMIO::Mapping* mmio, u32 base)
 {
+  constexpr u16 WMASK_NONE = 0x0000;
+  constexpr u16 WMASK_ALL = 0xffff;
+  constexpr u16 WMASK_LO_ALIGN_32BIT = 0xffe0;
+  const u16 WMASK_HI_RESTRICT = SConfig::GetInstance().bWii ? 0x1fff : 0x03ff;
+
   struct
   {
     u32 addr;
     u16* ptr;
     bool readonly;
-    bool writes_align_to_32_bytes;
+    // Some FIFO mmio regs only certain bits that always read as 0
+    // For _LO registers, this aligns to 32bits
+    // FOr _HI registers, some top bits are never set (different for wii and gcn)
```

Comment:
I believe this comment needs some love. The first line doesn't parse correctly in my head, as if some words are missing (although that might just be me as non-native speaker); and the last one has a capital letter O in "FOr".

Reply:
Can confirm that doesn't seem to read right.

Reply:
@BhaaLseN  Is that clearer?

Reply:
Better, thanks!